File: csharp/ScheduleTime.Domain/Entities/ServiceType/HairService.cs
namespace ScheduleTime.Domain.Entities.ServiceType
{
    public abstract class HairService
    {
        public bool Painting { get; set; }
        public string? Color { get; set; }
        public bool AnotherService { get; set; }
        public string? AnotherServiceText { get; set; }
        public DateTime CurrentDay { get; set; }
    }
}

File: java/schedule-time-event/src/main/java/com/gearguru/schedule_time_event/Domain/Entities/ServiceType/HairService.java
package com.gearguru.schedule_time_event.Domain.Entities.ServiceType;

import lombok.*;

import java.time.LocalDate;

@Getter(AccessLevel.PUBLIC)
@Setter(AccessLevel.PRIVATE)
@NoArgsConstructor
@AllArgsConstructor
public class HairService {
    private boolean painting;
    private String color;
    private boolean anotherService;
    private String anotherServiceText;
    private LocalDate currentDay;
}
